docs/actions.md

```diff
@@ -10,9 +10,9 @@ The following actions are used in NgrxJsonApi:
 | API_CREATE_INIT               | Issues a `POST` request for a resource.                                                                                                |
 | API_CREATE_SUCCESS            | Upon success completion of a `POST` request.                                                                                           |
 | API_CREATE_FAIL               | Upon a failure of a `POST` request.                                                                                                    |
```

Comment:
if we do renaming, one couldl create those API_POST_INIT, API_PATCH_INIT, etc. to be in sync with HTTP.